### PR TITLE
Display profile image in edit profile page

### PR DESCRIPTION
**Pull request by ervinj1998**
En esta rama se agrego la vista de la imagen que sube al usuario en la parte de editar perfil, sin embargo todavia no habilitamos la vista de la foto en la opcion principal de ver perfil, hasta la proxima

### CODE REVIEW

File: src/user/EditProfile.js [hunk 1]
```diff
@@ -213,6 +229,8 @@ clickSubmit = event => {
                            ""
                  )}
 
+                 <img src={photoUrl} alt={name} />
```
Comment (manzanarez83) [inline src/user/EditProfile.js:232]: la **_var photoUrl_** es con el objetivo de permitir subir una imagen ligada a la **_const photo Url_** la cual tambien define que si el usuario no tiene foto siempre mostrara un avatar por default.

**Comment by Hawaiideveloper** [comment]
@manzanarez83 where are he comments on the code and what it does?

File: src/user/EditProfile.js [hunk 2]
```diff
@@ -193,6 +203,12 @@ clickSubmit = event => {
         if(redirectToProfile){
             return <Redirect to = {`/user/${id}`} /> //se devuelve al perfil.
         }
+
+        const photoUrl = id ? `${process.env.REACT_APP_API_URL}/user/photo/${id}` //creamos la constante donde estara la foto,
+                            : DefaultProfile;  //creamos la constante donde estara la foto, si no dejamos la foto predeterminado, por si el usuario aun no tiene foto.
```
Comment (Hawaiideveloper) [inline src/user/EditProfile.js:208]: very good!

Comment (Hawaiideveloper) [reply src/user/EditProfile.js:208]: I like the explanation as it uses the `DefaultProfile` that was previously assigned an image

**Comment by Hawaiideveloper** [comment]
This is the most interesting branch as of to date.  I am now going to make some adjustments. After we do a pull

**Comment by Hawaiideveloper** [comment]
looks clean now doing a beta_version review and then will move to production folder

**Pull request merged**